### PR TITLE
Milestone 1 - Building with Responsive Design

### PR DESCRIPTION
**Pull request by alexandermorales-dev**
# TNW clone - milestone 1: Building with Responsive Design

In this milestones we: 

- Created the HTML skeleton for the web page and some basic styling with CSS. 
- Prepared the required assets from the original web page. 
- Applied different positioning techniques using Flex box, Floating elements and the Grid system. 
- Used Flex for Headers and Footers 
- Used Grid layout for Highlights and Latest Funding Sections 
- Applied styling on the different web page components to look similar to the original article. 
- Applied Media Queries for tablet and desktop screen sizes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,261 @@
...
+          <i class="fas fa-search search-icon"></i>
+          <i class="fas fa-bars menu-icon hidden-md hidden-lg"></i>
+        </span>
+      </div>
+      <div class="header-bottom-section">
+        <a href="#"
+          >Calling young digital talent in the Netherlands – Be recognized in
+          the T500 →</a
+        >
+      </div>
```
Comment (jcy2704) [inline index.html]: - [x] Use `<nav>` tag to enclose this section

File: index.html [hunk 2]
```diff
@@ -1,12 +1,261 @@
...
+            <li><a href="#">LATEST</a></li>
+            <li><a href="#">HARDFORKED</a></li>
+            <li><a href="#">PLUGGED</a></li>
+            <li><a href="#">README</a></li>
+            <li><a href="#">GROWTH QUARTERS</a></li>
+          </ul>
+        </section>
+        <span class="header-icons">
+          <i class="fas fa-search search-icon"></i>
+          <i class="fas fa-bars menu-icon hidden-md hidden-lg"></i>
```
Comment (jcy2704) [inline index.html]: - [x] Both should use `<a>` tags

File: index.html [hunk 3]
```diff
@@ -1,12 +1,261 @@
...
+        </ul>
+      </div>
+      <div class="header-top-section">
+        <section>
+          <img
+            alt="The Next Web"
+            class="black-bg random"
+            width="66"
+            src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg"
+          />
```
Comment (jcy2704) [inline index.html]: - [x] Logo needs to be enclosed in an `<a>` tag

File: index.html [hunk 4]
```diff
@@ -1,12 +1,261 @@
...
+          </h2>
+          <p class="secondary-text">
+            <span>by Ivan Mehta</span>
+            <i class="far fa-clock"></i>
+            1 hour ago
+            <i class="fas fa-share-alt"></i>
+            6
+          </p>
+        </div>
+      </article>
```
Comment (jcy2704) [inline index.html]: - [x] Each article should use an `<a>` tag

File: index.html [hunk 5]
```diff
@@ -1,12 +1,261 @@
...
+          <i class="far fa-envelope"></i>
+        </div>
+        <div class="bottom-links">
+          <a href="#">EVENTS</a>
+          <a href="#">ABOUT</a>
+          <a href="#">ADVERTISE</a>
+          <a href="#">JOBS</a>
+          <a href="#">CONTACT</a>
+        </div>
+        <div></div>
```
Comment (jcy2704) [inline index.html]: - [x] If not using this div, please remove it.

File: index.html [hunk 6]
```diff
@@ -1,12 +1,261 @@
...
+          <i class="fas fa-rss"></i>
+          <i class="far fa-envelope"></i>
+        </div>
+        <div class="bottom-links">
+          <a href="#">EVENTS</a>
+          <a href="#">ABOUT</a>
+          <a href="#">ADVERTISE</a>
+          <a href="#">JOBS</a>
+          <a href="#">CONTACT</a>
+        </div>
```
Comment (jcy2704) [inline index.html:283]: - [ ] [OPTIONAL] It can use `<ul>` and `<li>` tags

**Review by jcy2704 — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 👀🤔

## Hey @AlexJustAlex2020  @omar-labana!! I see you've worked hard on this project. 😏

### However, there are some things that we need to change and improve before your project be approved.

### First, let's talk about your README

- **Really professional! I like how you use the template that was offered. However, there are some things we need to add to the README to make it 11/10.**

  - [x] Add more description to the project. Remove the "> Project 2: Buil...." part and add a brief description of the project and the work you've done.
  
  - [x] The screenshot must be from your project and not from the original.

### Now, your project

- **You've done a really good job, there are many parts of your project that look similar to the original one so point for that, but it doesn't look completely similar to the original so we need to improve that.**

|Original|Clone|
|-------|------|
| image | image |

  - [x] Remove logo in desktop resolution
  - [x] Align correctly the items in the top nav
  - [x] Align correctly the text in the yellow container

<hr>

|Original|Clone|
|-------|------|
| image | image |

- [x] Align correctly the items in the top nav
- [x] Remove hamburger menu in desktop resolution

<hr>

|Original|Clone|
|-------|------|
| image | image |

- [x] Missing the TNW logo
- [x] Text is smaller in desktop resolution compared to the original
- [x] Articles are smaller compared to the original
- [x] Add the gradient in the main article

<hr>

|Original|Clone|
|-------|------|
| image| image |

- [x] Remove top nav from mobile resolution
- [x] Text in the yellow container is bigger than the original
- [x] Articles are smaller than the original
- [x] Search and Hamburger icons are bigger than the original

<hr>

|Original|Clone|
|-------|------|
| image| image |

- [x] Navbar must be at the top when scrolling down
- [x] Align all the items correctly on each card
- [x] Some text is bigger than the original

<hr>

|Original|Clone|
|-------|------|
| image| image |

- [x] Align the items in the footer correctly

### SEE BELOW FOR MORE CHANGES 👇

<hr>

### Keep the awesome work! 💪 Happy Coding for y'all!! 💻

**If you have any questions you can ask me on my Slack.**

**Best,**

**[💁‍♂️ Steven Jack Chung Yau](https://stevenchung.me)**

[LinkedIn](https://linkedin.com/in/stevenjchung) | [Twitter](https://twitter.com/_yiak) | [GitHub](https://github.com/jcy2704)

**Review by elisha2kyakpo1 — APPROVED** [review]
💪**Congratulations guys let's celebrate, I'm happy to see you implement all the changes requested**
[OPTIONAL] your big article is missing a date, you can fix that in your free time.
Cheers
Elisha kyakopo, we can connect [Twitter](https://twitter.com/Elisha1k)

**Pull request closed without merging**